### PR TITLE
Modify quantile method to provide smoother results

### PR DESCRIPTION
**Pull request by Rlamboll**
# Pull request

Please confirm that this pull request has done the following:

- [x] Tests added
- [x] Documentation added (where applicable)
- [x] Example added (either to an existing notebook or as a new notebook, where applicable)
- [x] Description in ``CHANGELOG.rst`` added

## Adding to CHANGELOG.rst

Please add a single line in the changelog notes similar to one of the following:

```
- (`#XX <https://github.com/znicholls/silicone/pull/XX>`_) Added feature which does something
- (`#XX <https://github.com/znicholls/silicone/pull/XX>`_) Fixed bug identified in (`#YY <https://github.com/znicholls/silicone/issues/YY>`_)
```

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,6 +4,7 @@ Changelog
 master
 ------
 
+- (`#78 <https://github.com/znicholls/silicone/pull/78>`_) Changed how quantile rolling windows works by adding an extra interpolate step for smoothness
```
Comment (znicholls) [inline CHANGELOG.rst:7]:
```suggestion
- (`#78 <https://github.com/znicholls/silicone/pull/78>`_) Updated ``DatabaseCruncherQuantileRollingWindows `` to use an extra interpolate step for smoothness
```

File: scripts/compare_crunchers.py [hunk 2]
```diff
@@ -28,7 +28,7 @@ def main():
     crunchers_list = [
         #  dc.DatabaseCruncherLeadGas,
         # dc.DatabaseCruncherTimeDepRatio,
-        dc.DatabaseCruncherQuantileRollingWindows,
+        dc.QuantileRollingWindows,
```
Comment (znicholls) [inline scripts/compare_crunchers.py:31]: Why the change to only one class?

Comment (Rlamboll) [reply scripts/compare_crunchers.py:31]: I'll do the rest in the next release but since I have to re-run everything using this cruncher here I thought I'd split the problem up

File: CHANGELOG.rst (hunk 1 above)
Comment (Rlamboll) [reply CHANGELOG.rst:7]: The maths is super-fiddly, it should be ok though.

File: scripts/compare_crunchers.py (hunk 2 above)
Comment (znicholls) [reply scripts/compare_crunchers.py:31]: Also works, can you make a quick issue to remind us to do the rest?
